pkg/clusterconfig/clusterconfig.go

```diff
@@ -111,22 +107,22 @@ func GetAWSConfig() (*Config, error) {
 	}
 
 	// Look for a user defined secret to get the AWS credentials from first
-	sec, err := client.Secrets(imageregistryv1.ImageRegistryOperatorNamespace).Get(imageregistryv1.ImageRegistryPrivateConfigurationUser, metav1.GetOptions{})
+	sec, err := listers.Secrets.Get(imageregistryv1.ImageRegistryPrivateConfigurationUser)
 	if err != nil && errors.IsNotFound(err) {
 		// If no user defined secret is found, use the system one
-		sec, err = client.Secrets(installerConfigNamespace).Get(installerAWSCredsName, metav1.GetOptions{})
+		sec, err = listers.InstallerSecrets.Get(installerAWSCredsName)
 		if err != nil {
-			return nil, fmt.Errorf("unable to get secret %q: %v", fmt.Sprintf("%s/%s", installerConfigNamespace, installerAWSCredsName), err)
+			return nil, fmt.Errorf("unable to get secret %q: %v", fmt.Sprintf("%s/%s", "kube-system", installerAWSCredsName), err)
```

Comment:
why is this moving away from being a constant?

Reply:
leftover from another change I had tested.  Fixed in next push